gateway-service/src/main/resources/gateway-api-doc.json

```diff
@@ -65,7 +69,7 @@
                 }
             }
         },
-        "/query": {
+        "/auth/query": {
```

Comment:
This was not changed by you but we have typo here
![image](https://user-images.githubusercontent.com/6776217/76195726-3a344600-61e9-11ea-8063-b46af8c4bcfe.png)

Reply:
Fixed.